recipe/meta.yaml

```diff
@@ -18,14 +18,14 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
-    - cudatoolkit-dev   {{ cudatoolkit }}
-    - make {{ make }}  
+    - nvcc  {{ cudatoolkit }}
```

Comment:
I think we need to add this as `- {{ compiler("cuda") }}` and then have the following in conda config:

```
cuda_compiler:
- nvcc
```

Reply:
Do we also need   

```
script_env:
    - CUDA_HOME
```
in the build section?

Reply:
Yeah, I like the approach of adding the `cuda_compiler` to conda_build_config.

Did a build work without adding `CUDA_HOME` to `script_env`? It would be good to understand if it is necessary.

Reply:
A build did work without the `script_env` section, but I'll explore a bit more.

Reply:
`script_env` is needed. Or at least it's more proper. I've added it back.